blaze/expr/expressions.py

```diff
@@ -22,7 +22,7 @@
 __all__ = ['Expr', 'ElemWise', 'Field', 'Symbol', 'discover', 'Projection',
            'projection', 'Selection', 'selection', 'Label', 'label', 'Map',
            'ReLabel', 'relabel', 'Apply', 'Slice', 'shape', 'ndim', 'label',
-           'symbol']
+           'symbol', 'Coerce', 'coerce']
```

Comment:
Should the `coerce` function be available as a standalone function from the top level package or do you think this is most useful as a method